end_to_end/0-AutoClaimFraudDetection.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<p>The plot is too crowded. Increase size might help to make space for the labels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>